README.md

```diff
@@ -722,6 +722,62 @@ Req/Bytes counts sampled once per second.
 125k requests in 10.03s, 43.7 MB read
 ```
 
+## More info about how this plugin works
+This plugin caches the resolver's results. The plugin has nothing to do with validating the schema, so if you have a resolver that returns a `String` but in your schema it indicates that the resolver returns an `Int`, the plugin will cache the `String`, and when you call the resolver again, the plugin will return the cached `String` instead of the `Int`. 
+As a result, the query will return a Schema Validation error and not the right result.
+
+
+Here you can find an example of the problem.
+```js
+'use strict'
+
+const fastify = require('fastify')
+const mercurius = require('mercurius')
+const cache = require('mercurius-cache')
+
+const app = fastify({ logger: true })
+
+const schema = `
+  type Query {
+    add(x: Int, y: Int): Int  // <-- this is a Int
```

Comment:
I created a simple example with a `getNumber` solver.
I cannot use the `hello` resolver because if I set the return type as `String` and then return an `Int`, everything works smoothly because the number converts to a `String`.
I created a `getNumber` to make sense of the example.